inabox/deploy/config.go

```diff
@@ -197,30 +197,31 @@ func (env *Config) generateDisperserVars(ind int, key, address, logPath, dbPath,
 // Generates batcher .env
 func (env *Config) generateBatcherVars(ind int, key, graphUrl, logPath string) BatcherVars {
 	v := BatcherVars{
-		BATCHER_S3_BUCKET_NAME:              "test-eigenda-blobstore",
-		BATCHER_DYNAMODB_TABLE_NAME:         "test-BlobMetadata",
-		BATCHER_ENABLE_METRICS:              "true",
-		BATCHER_METRICS_HTTP_PORT:           "9094",
-		BATCHER_PULL_INTERVAL:               "5s",
-		BATCHER_BLS_OPERATOR_STATE_RETRIVER: env.EigenDA.OperatorStateRetreiver,
-		BATCHER_EIGENDA_SERVICE_MANAGER:     env.EigenDA.ServiceManager,
-		BATCHER_SRS_ORDER:                   "300000",
-		BATCHER_CHAIN_RPC:                   "",
-		BATCHER_PRIVATE_KEY:                 key[2:],
-		BATCHER_STD_LOG_LEVEL:               "debug",
-		BATCHER_FILE_LOG_LEVEL:              "trace",
-		BATCHER_LOG_PATH:                    logPath,
-		BATCHER_GRAPH_URL:                   graphUrl,
-		BATCHER_USE_GRAPH:                   "true",
-		BATCHER_BATCH_SIZE_LIMIT:            "10240", // 10 GiB
-		BATCHER_INDEXER_PULL_INTERVAL:       "1s",
-		BATCHER_AWS_REGION:                  "",
-		BATCHER_AWS_ACCESS_KEY_ID:           "",
-		BATCHER_AWS_SECRET_ACCESS_KEY:       "",
-		BATCHER_AWS_ENDPOINT_URL:            "",
-		BATCHER_FINALIZER_INTERVAL:          "6m",
-		BATCHER_ENCODING_REQUEST_QUEUE_SIZE: "500",
-		BATCHER_NUM_CONFIRMATIONS:           "0",
+		BATCHER_S3_BUCKET_NAME:                "test-eigenda-blobstore",
+		BATCHER_DYNAMODB_TABLE_NAME:           "test-BlobMetadata",
+		BATCHER_ENABLE_METRICS:                "true",
+		BATCHER_METRICS_HTTP_PORT:             "9094",
+		BATCHER_PULL_INTERVAL:                 "5s",
+		BATCHER_BLS_OPERATOR_STATE_RETRIVER:   env.EigenDA.OperatorStateRetreiver,
+		BATCHER_EIGENDA_SERVICE_MANAGER:       env.EigenDA.ServiceManager,
+		BATCHER_SRS_ORDER:                     "300000",
+		BATCHER_CHAIN_RPC:                     "",
+		BATCHER_PRIVATE_KEY:                   key[2:],
+		BATCHER_STD_LOG_LEVEL:                 "debug",
+		BATCHER_FILE_LOG_LEVEL:                "trace",
+		BATCHER_LOG_PATH:                      logPath,
+		BATCHER_GRAPH_URL:                     graphUrl,
+		BATCHER_USE_GRAPH:                     "true",
+		BATCHER_BATCH_SIZE_LIMIT:              "10240", // 10 GiB
+		BATCHER_INDEXER_PULL_INTERVAL:         "1s",
+		BATCHER_AWS_REGION:                    "",
+		BATCHER_AWS_ACCESS_KEY_ID:             "",
+		BATCHER_AWS_SECRET_ACCESS_KEY:         "",
+		BATCHER_AWS_ENDPOINT_URL:              "",
+		BATCHER_FINALIZER_INTERVAL:            "6m",
+		BATCHER_ENCODING_REQUEST_QUEUE_SIZE:   "500",
+		BATCHER_NUM_CONFIRMATIONS:             "0",
+		BATCHER_MAX_BLOBS_TO_FETCH_FROM_STORE: "1",
```

Comment:
why set it to 1?
this may slow down testing if we disperse more than 1 blob at a time unless it's explicitly testing a scenario that requires multiple fetches